src/AssemblerReadGraph2.cpp

```diff
@@ -4,11 +4,143 @@ using namespace shasta;
 
 
 void Assembler::createReadGraph2(
-    uint32_t maxAlignmentCount,
-    uint32_t maxTrim)
+    uint32_t maxAlignmentCount)
 {
-    // This boilerplate code creates the read graph using
-    // all available alignments.
-    vector<bool> keepAlignment(alignmentData.size(), true);
+    // Percentile cutoffs for automated parameters
+    double largePercentile = 0.12;
```

Comment:
Are these the only two parameters? Do we the flexibility to allow need different percentiles for different quantities?

Reply:
I was initially trying not to add more parameters, but the more I think about it, the more it makes sense to have the option to change them independently if needed